lua/entities/gmod_wire_egp/lib/egplib/transmitreceive.lua

```diff
@@ -358,8 +358,10 @@ if (SERVER) then
 				E2.prf = E2.prf + 10
 			end
 
-			EGP:SaveFrame( E2.player, Ent, Data[1] )
-			self:AddQueue( Ent, E2.player, SaveFrame, "SaveFrame", Data[1] )
+			local status = EGP:SaveFrame( E2.player, Ent, Data[1] )
```

Comment:
adding an intermediate variable can help people understand the code better, but in this case, "status" isn't saying much, so idk, you could just do `if EGP:SaveFrame(...) then`.
Your call.

Reply:
local's gone now